### PR TITLE
[All] Various small fixes

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: Sources/Base/Values.swift [hunk 1]
```diff
@@ -120,19 +124,16 @@ class FloatExpr: ConstantExpr {
...
-  let raw: String
-  init(value: Double, raw: String, sourceRange: SourceRange? = nil) {
+  init(value: Double, sourceRange: SourceRange? = nil) {
     self.value = value
-    self.raw = raw
     super.init(sourceRange: sourceRange)
   }
   override var text: String {
-    return raw
+    return "\(value)"
```
Comment (segiddins) [inline Sources/Base/Values.swift:132]: `value.description`?

File: Sources/Parse/Lexer.swift [hunk 2]
```diff
@@ -125,8 +125,8 @@ enum TokenKind: Equatable {
   
   var text: String {
     switch self {
-    case .number(let value): return "\(value)"
-    case .float(let left, let right, _): return "\(left).\(right)"
+    case .number(_, let raw): return raw
+    case .float(let value): return "\(value)"
```
Comment (segiddins) [inline Sources/Parse/Lexer.swift:129]: `value.description`

**Review by segiddins — APPROVED** [review]
Looks really good!

**Comment by harlanhaskins** [comment]
Merging because CI passed before.

**Pull request merged**